coremain/version.go

```diff
@@ -2,7 +2,7 @@ package coremain
 
 const (
 	coreName    = "CoreDNS"
-	coreVersion = "003"
```

Comment:
Think it is OK. Just surprised this shows up as a diff after a rebase, but that maybe me not understanding git well enough.